### DISCUSSION TITLE
The 'uk-' prefix is a turnoff - atleast for me

### DISCUSSION THREAD

**Discussion by ibnsultan** [franken-ui/ui#36] [General]
Hi there folks, you guys did a wonderfull job but i gotta ask, was `uk-` prefix really necessary at the beginning of every class name, speaking from my xp only, it feels very unnatural, i started a project with franken ui and now whenever I look at its codebase the overflowing of the uk-  everywhere `uk-this`, `uk-there`, like comeon.

Maybe you guys have a reason for this?
Regard

**Comment by eydun** [comment]
Franken UI is based upon: tailwindcss, shadcn and Uikit (https://getuikit.com). The "uk-" prefix is from Uikit. Consider it is a namespace for css-classes, so that they do not collide with classes from other packages.

**Reply by ibnsultan** [reply]
oh that makes sense now, thanks
